big_tests/tests/sasl_external_SUITE.erl

```diff
@@ -98,6 +121,10 @@ verify_mode_by_group_name(fast_tls) ->
     "";
 verify_mode_by_group_name(just_tls) ->
     "";
+verify_mode_by_group_name(fast_tls_no_common_name) ->
+    "";
+verify_mode_by_group_name(just_tls_no_common_name) ->
+    "";
```

Comment:
How about moving 
```erlang
verify_mode_by_group_name(_) -> "".
```
as a last match?

Reply:
I wouldn't do that, cause we might want to add more groups which might require different configs. This way it is clearer in my opinion.

Reply:
Seems legit